src/model/mod.rs

```diff
@@ -48,8 +50,145 @@ pub enum PlayingItem {
     Episode(show::FullEpisode),
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct Id<'id> {
```

Comment:
I am thinking about that is `Id` a good name to describe its functionality? and it's same to `from_id_or_uri`

Reply:
The `Id` type represents spotify object id, so yes, I think it's the most correct name for it.
Other possible names are `SpotifyId` (it's a little bit verbose, as we are already in a crate named `rspotify`), `Uri`, or `SpotifyUri` (too verbose, as well as `SpotifyId`).

Reply:
Do you have better proposals?

Reply:
I don't see how `Id` isn't an appropiate name either. `SpotifyId` is redundant.